README.md

```diff
@@ -96,7 +112,7 @@ The input file is a JSON file in the format:
 - Example complete url: "https://api.github.com/repos/spring-projects/spring-framework": 19805,
```

Comment:
This line and the one after should have backticks around the content that would actually appear in a config file.

Reply:
added [backticks](https://github.com/indeedeng/Mariner/pull/111/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5)